src/dune_rules/mdx.ml

```diff
@@ -145,13 +145,20 @@ module Prelude = struct
     in
     enter decode_env <|> decode_default
 
+  let bpath ~dir p = Path.build (Path.Build.append_local dir p)
```

Comment:
Let's inline this function. It doesn't have a good name nor any logic but it exists in a much larger scope.